docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+
+### Exceptions
+When an enhancement fails to meet the release deadline, Enhancement owner must file an
+ [exception request](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md#requesting-an-exception)
+ addressed to SIG sponsoring the enhancement, the Release Team, and SIG Release to gain approval.
+ The release team will be responsible for approving or rejecting exceptions and the decision
+ will be based on discussion with the sponsoring SIG's chairs/leads and any other participating
+ SIG's chairs/leads.
+
+More details can be found in [Exceptions to Milestone Enhancement Complete dates](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md).
+
+### Enhancements Freeze
+All enhancements going into the release must meet the following criteria by the enhancements freeze deadline:
+- An open issue in the k/enhancements repo
+- A KEP in an `implementable` state
+- A KEP with test plans
+- A KEP with graduation criteria
+- A KEP with production readiness review approved
+
+Any enhancements that does not meet all the criteria will be removed from tracking for the upcoming release.
+ Any unmerged pull requests in k/enhancements repo for the upcoming release milestone will be removed
+ by the Release Team and will require an approved exception to have the milestone added back and the enhancement
+ tracked again by the Release Team.
+
+More details can be found in [Enhancements Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#enhancements-freeze).
+
+### Kubernetes Enhancement Proposal (KEP)
+A Kubernetes Enhancement Proposal is a way to propose, communicate and coordinate on new efforts
+ for the Kubernetes project.
+
+A KEP should be created after socializing an idea with the sponsoring and participating SIGs.
+ To create a KEP, the [KEP template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md)
+ should be used and follow the process outlined in the template.
+
+Use the [definition of what is an Enhancement](https://github.com/kubernetes/enhancements#is-my-thing-an-enhancement)
+ to determine if a KEP is required.
+
+More details can be found in [Kubernetes Enhancement Proposals](https://github.com/kubernetes/enhancements/blob/master/keps/README.md#kubernetes-enhancement-proposals-keps)
+ and [Enhancements](https://github.com/kubernetes/enhancements/blob/master/README.md).
+
+### Production Readiness Review (PRR)
+Production readiness reviews are intended to ensure that features merging into Kubernetes
+ are observable, scalable and supportable, can be safely operated in production environments,
+ and can be disabled or rolled back in the event they cause increased failures in production.
+
+PRR approval is a requirement for the enhancement to be part of the upcoming release.
+ In addition to answering PRR related questions in the KEP template, an Enhancement owner
```

Comment:
would prefer to directly link to PRR page specifically here: https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval instead of having directions that may go stale